master/internal/api_user.go

```diff
@@ -352,18 +395,16 @@ func (a *apiServer) PatchUser(
 			return nil, status.Error(codes.PermissionDenied, err.Error())
 		}
 
-		u := &userv1.User{}
-
 		displayName, err2 := clearUsername(targetUser, *req.User.DisplayName, 0)
 		if err2 != nil {
 			return nil, err2
 		}
 
 		if displayName == nil || *displayName == "" {
 			// Set empty display name to null.
-			err = a.m.db.QueryProto("set_user_display_name", u, req.UserId, nil)
+			err = user.SetDisplayName(req.UserId, nil)
 		} else {
-			err = a.m.db.QueryProto("set_user_display_name", u, req.UserId, displayName)
+			err = user.SetDisplayName(req.UserId, displayName)
```

Comment:
could you coordinate this with @NicholasBlaskey re: https://determinedai.atlassian.net/browse/DET-8659